release_config.yml

```diff
@@ -0,0 +1,13 @@
+release-platforms:
+  - GitHub
+  - Maven
+  - Community
+community-tags:
+  - Virtual Schema
+  - Exasol
+  - Open Source
+  - GitHub
+community-project-name: Exasol Virtual Schema
+community-project-description: |
+  The Virtual Schema provides an abstraction layer that makes an external database accessible from an Exasol database through regular SQL commands.
+  The contents of the external database are mapped to virtual tables which look like, and can be queried as, any regular Exasol table.
```

Comment:
I think you should mention here that this is the virtual schema for an external exasol database. To me the description sounded like a generic description of the virtual-schema concept.